glossary.yml

```diff
@@ -1155,6 +1155,14 @@
       Ni thamani halisi inayopitishwa. Vigezo na hoja ni tofauti, lakini dhana zinazohusiana. 
       Vigezo ni vigeuzo na hoja ni thamani zilizopewa vigeu hivyo.
 
+  uk:
```

Comment:
@Dod900ls1 It is acceptable, but in the future don't add blank lines - only different slugs should be separated by a new line, but different translations within a slug need not to be separated by blank lines 

Since you know now how to edit PRs, you can also use this as an exercise :)